src/core/builder.ts

```diff
@@ -46,7 +46,7 @@ class DepGraphBuilder {
     this._pkgManager = pkgManager;
   }
 
-  // TODO: this can create disconnected nodes
+  // TODO: this can create disconnected nodes -- and memory leaks!
```

Comment:
how?

Reply:
added this note prior to seeing the underlying `graph` object